tests/config/new_op_r.cfg

```diff
@@ -1,3 +1,4 @@
-sp_between_new_paren=remove
-sp_after_newop_paren=remove
-sp_inside_newop_paren=remove
+sp_between_new_paren  = remove
+sp_after_newop_paren  = remove
+sp_inside_newop_paren = remove
+sp_paren_paren        = force
```

Comment:
Having the code for the option `sp_paren_paren` before (as proposed) we get the value `ignore`.
Which is a different run of uncrustify.
We need to define what we want to have for `((` and `))`.

Reply:
I still don't quiet understand yet why this is necessary, I compile it and see for myself. It would be nice if you could remove this commit from #1350 to keep everything separated.